cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -1060,10 +1076,8 @@ def shutdown():
     """ Stop the kubernetes master services
 
     """
-    service_stop('snap.kube-apiserver.daemon')
-    service_stop('snap.kube-controller-manager.daemon')
-    service_stop('snap.kube-scheduler.daemon')
-    service_stop('snap.kube-proxy.daemon')
+    for service in master_services:
+        service_stop('snap.%s.daemon' % service)
```

Comment:
It originally said snap.*.daemon and I am happy with this, but the service_stop comments claim %s only. We should clarify the called function or maybe I was just looking at the wrong thing as it was initd stuff.

Reply:
This is calling `charmhelpers.core.host.service_stop`, which calls `systemctl stop <service>`, which complains if you omit the `snap.` and `.daemon` bits:
```
$ sudo systemctl stop kube-proxy
Failed to stop kube-proxy.service: Unit kube-proxy.service not loaded.
```

I'm pretty sure we need it.